### PR TITLE
Add additional foreign national terms to glossary

### PR DESCRIPTION
**Pull request by djgarr**
## Summary (required)

Add additional foreign national terms to the glossary https://docs.google.com/spreadsheets/d/1iSHhncHt8JCzYrh_UEHmyhoy3sKEpAnNNspCYdS188g/edit#gid=0 (see highlighted)

- Resolves (https://github.com/fecgov/fec-cms/issues/5042)

### CODE REVIEW

File: fec_eregs/static/fec_eregs/js/data/terms.json [hunk 1]
```diff
@@ -318,6 +318,10 @@
   {
     "term": "Labor organization",
     "definition": "An organization, agency or employee representative committee or plan, in which employees participate and which exists for the purpose of dealing with employers on grievances, labor disputes, wages, hours of employment or working conditions. 11 CFR <a href=\"https://www.fec.gov/regulations/114-1/CURRENT#114-1-d\">114.1(d)</a>."
+  },
+      {
+    "term": "Lawfully admitted for permanent residence",
+    "definition": "An individual lawfully admitted for permanent residence is someone who has been granted authorization to live and work in the United States on a permanent basis. As proof of that status, U.S. Citizenship and Immigration Services (USCIS) grants a person a permanent resident card, commonly called a "Green Card.""
```
Comment (patphongs) [inline fec_eregs/static/fec_eregs/js/data/terms.json:324]: @djgarr When there are quotations as part of the text, it needs to be escaped with a backwarrds slash. Like this `...called a \"Green Card.\""`. After that change, it'll work, thanks!

**Review by patphongs — CHANGES_REQUESTED** [review]
One minor change

**Review by dorothyyeager — APPROVED** [review]
Approved from content team perspective. Thanks for taking this on David, nice job!

**Review by patphongs — APPROVED** [review]
All good, thanks again @djgarr !

**Pull request merged**